app/controllers/admin/appointments_controller.rb

```diff
@@ -77,19 +77,19 @@ def load_appointment
     end
 
     helper_method def items_available_to_add_to_pickup
-      Item.available.eager_load(:borrow_policy)
+      Item.available.includes(:borrow_policy)
```

Comment:
I've got a separate PR trying to address that this returns nearly the whole item collection to render into that dropdown, which I'm suspicious is also contributing to page slowness.

Reply:
Definitely suspicious